### DISCUSSION TITLE
Is there a way to restrict content by Roles?

### DISCUSSION THREAD

**Discussion by placeholdergit** [decaporg/decap-cms#6216] [General]
I can only set the roles in Netlify CMS GUI but cannot see anyway to filter it so that the role of that user would not be able to edit or modify other roles post or something similar.

**Comment by kuwts** [comment]
Did you ever end up figuring this out?

**Comment by martinjagodic** [comment]
Unfortunately, roles are not a feature of Decap. There is a workaround if you use Netlify Identity. Create a new Netlify site for each role and have each site use a specific config.yml. Then you can invite users for each role from its respective site. The downside is that users on different roles will require different links to access the CMS.
